### ISSUE TITLE
Example of large gap jumping being required in chrome but not firefox.

### ISSUE THREAD

**Issue by TheJohnBowers** [google/shaka-player#1225]
**Have you read the [FAQ](https://goo.gl/JE1Sy5) and checked for duplicate issues**:
Yes
**What version of Shaka Player are you using**:
2.3.0 built from master which I updated from git today.
**Can you reproduce the issue with our latest release version**:
I did not attempt it, but my version of master old and reported 2.2 build before I updated it today, and it reproduced before and after my update.
**Can you reproduce the issue with the latest code from `master`**:
Yes
**Are you using the demo app or your own custom app**:
Demo app
**If custom app, can you reproduce the issue using our demo app**:
NA
**What browser and OS are you using**:
Mac OS 10.12.6
**What are the manifest and license server URIs**:
*(you can send the URIs to <shaka-player-issues@google.com> instead, but please use GitHub and the template for the rest)*

```js
  {
    name: 'Playback Hang Exiting First Ad Break',
    manifestUri: 'https://content.uplynk.com/c6236a686e45449c9870e081697a1df3.mpd?ad=problem', // gjslint: disable=110
    encoder: shakaAssets.Encoder.UPLYNK,
    source: shakaAssets.Source.UPLYNK,
    drm: [
      shakaAssets.KeySystem.PLAYREADY,
      shakaAssets.KeySystem.WIDEVINE
    ],
    features: [
      shakaAssets.Feature.MP4,
      shakaAssets.Feature.PSSH,
      shakaAssets.Feature.MULTIKEY,
      shakaAssets.Feature.AESCTR_8_BYTE_IV,
      shakaAssets.Feature.SEGMENT_LIST_DURATION,
      shakaAssets.Feature.HIGH_DEFINITION
    ],
    licenseServers: {
      'com.microsoft.playready': 'https://content.uplynk.com/pr',
      'com.widevine.alpha': 'https://content.uplynk.com/wv'
    },
    requestFilter: shakaAssets.UplynkRequestFilter,
    responseFilter: shakaAssets.UplynkResponseFilter
  },
```

**What did you do?**

Playback a multi-period asset with a newly encoded test ad.

**What did you expect to happen?**

I expected it to playback without hanging

**What actually happened?**

So the reason I am filing this bug is because there is a difference in behavior using the same version of shaka on chrome and firefox.

On chrome I notice a hang exiting the second period at second 224 approximately.  Chrome seems to think there is a "large gap" here, and won't play back unless I enable large gap jumping, but then it actually skips a few seconds of the third period.  

Firefox doesn't seem to notice anything but a very small gap, which it ignores and plays through without skipping any video.  

From firefox I see:  

Jumping forward 0.10010799999997744 seconds because of gap starting at 224.828583 and ending at 224.89

From Chrome I see:

Ignoring large gap at 224.969779  playhead.js:527 
Ignoring large gap at 227.755557

**Comment by joeyparrish** [comment]
I tried this in Chrome & Chrome beta, the latter using an experimental flag that changes MSE behavior with regard to timestamps.

Unfortunately, I got the same results in both instances.  What surprised me was that while I was paying attention to the beta instance, the copy of Shaka running in Chrome stable eventually decided to jump the gap after all...

```js
Ignoring large gap at 224.939685 playhead.js:527
Ignoring large gap at 227.225531 playhead.js:527

// ... time passes ...

Ignoring large gap at 227.343401 playhead.js:527
Jumping forward 0.3898969999999906 seconds because of gap starting at 224.828583 and ending at 228.985999 playhead.js:536
```

We will continue to look into this.  Can we count on this asset remaining up in the short term while we investigate?  If this ends up as a Chrome bug, can we keep this asset available long-term to help validate the fix and watch for regressions?

**Comment by TheJohnBowers** [comment]
Yes, I will leave that asset be for as long as you need it.

**Comment by TheModMaker** [comment]
I believe this was fixed by 87c19da2c, can you please try the latest `master`.

**Comment by TheJohnBowers** [comment]
Ok, Looks good to me.  I tried it in the 2.2 and 2.3 branches and issue seems to be gone.  Thanks for looking into it.
